src/plugins/ipaddr/README.md

```diff
@@ -0,0 +1,46 @@
+- infos = Information about the ipaddr plugin is in keys below
+- infos/author = Author Name <elektra@libelektra.org>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides =
+- infos/recommends =
+- infos/placements =
+- infos/status = recommended productive maintained reviewed conformant compatible coverage specific unittest shelltest tested nodep libc configurable final preview memleak experimental difficult unfinished old nodoc concept orphan obsolete discouraged -1000000
+- infos/metadata =
```

Comment:
Missing. Please be compatible with metadata of `network` plugin. (See also below)